restaurants/src/app/components/restaurantdetails/restaurantdetails.component.html

```diff
@@ -0,0 +1,13 @@
+<div *ngIf="restaurant">
+    <h2>{{restaurant.name | uppercase}} Details</h2>
+    <div>
+      <label for="restaurant-name">Restaurant name: </label> 
+        {{restaurant.name}}
+    </div>
+    <div><span>rank: </span>{{restaurant.rank}}</div>
+    <button type="button" (click)="delete()">Delete Restaurant
+        <a routerLink ="/restaurants"></a>
+    </button>
+
+    <button type="button" (click)="goBack()">go back</button>
```

Comment:
would be nice to add ids to both of the buttons - and anywhere else that you have buttons and inputs. Not a big deal if they're missing now, but can make things easier down the line.

Reply:
add for those two